docs/source/traits_user_manual/listening.rst

```diff
@@ -0,0 +1,782 @@
+.. _on-trait-change-notification:
+
+===========================================
+Trait Notification with ``on_trait_change``
+===========================================
+
+Requesting trait attribute change notifications can be done in several
+ways:
+
+.. index:: notification; strategies
+
+* Dynamically, by calling on_trait_change() or on_trait_event() to establish
+  (or remove) change notification handlers.
+* Statically, by decorating methods on the class with the @on_trait_change
+  decorator to indicate that they handle notification for specified attributes.
+* Statically, by using a special naming convention for methods on the class to
+  indicate that they handle notifications for specific trait attributes.
+
+.. index:: notification; dynamic
```

Comment:
This probably needs to be updated to give the context for the older methods of notification.